Week2/homework/maartjes_work.js

```diff
@@ -41,7 +39,10 @@ const tuesday = [
     duration: 40
   }
 ];
-
 const tasks = monday.concat(tuesday);
-
-// Add your code here
+let twoDaysIncome = tasks
+  .map(task => task.duration / 60)
+  .filter(task => task >= 2)
+  .map(total => total * 60)
+  .reduce((a, b) => a + b, 0);
+console.log("Maartje earned " + twoDaysIncome + ".00 Euro on Monday and Tuesday!");
```

Comment:
For this last part, you were supposed to round the amount to 2 decimals. Let's imagine that Maartje makes $59/hr instead of 60. You cannot be sure that it will always be an exact amount. Javascript has a functionality just for this use-case: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed 

Sample:
```javascript
console.log("Maartje earned " + twoDaysIncome.toFixed(2) + " Euro on Monday and Tuesday!"); 
```